subscriptions/providers/apple_in_app/__init__.py

```diff
@@ -85,7 +84,6 @@ def webhook(self, request: Request, payload: dict) -> Response:
         for request_class, handler in handlers.items():
             with suppress(ValidationError):
                 instance = request_class.parse_obj(payload)
```

Comment:
Just figured out that this won't show what validation error happened in bad case, and it will be hard to debug. Yeah, I remember this code was my idea, but it turns out to be unusable. Maybe have a look at same piece of code in google subscriptions. Or maybe revert to checking some "notification-specific" fields as you did earlier. Sorry, my fault

Reply:
I believe that I have something in mind.

Reply:
Example log:
```
Failed matching the payload to any registered request: [
  {
    "loc": [
      "transaction_receipt"
    ],
    "msg": "field required",
    "type": "value_error.missing"
  },
  {
    "loc": [
      "test"
    ],
    "msg": "extra fields not permitted",
    "type": "value_error.extra"
  }
]
[
  {
    "loc": [
      "signedPayload"
    ],
    "msg": "field required",
    "type": "value_error.missing"
  },
  {
    "loc": [
      "test"
    ],
    "msg": "extra fields not permitted",
    "type": "value_error.extra"
  }
].
```

Reply:
```
Failed matching the payload to any registered request:
2 validation errors for AppleReceiptRequest
transaction_receipt
  field required (type=value_error.missing)
test
  extra fields not permitted (type=value_error.extra)

2 validation errors for AppleAppStoreNotification
signedPayload
  field required (type=value_error.missing)
test
  extra fields not permitted (type=value_error.extra).
```
Easier for the eyes.